pom.xml

```diff
@@ -16,47 +16,47 @@
         <sonar.coverage.jacoco.xmlReportPaths>../multiapps-controller-coverage/target/site/jacoco-aggregate/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
         <junit.version>5.10.2</junit.version>
         <junit-platform.version>1.10.2</junit-platform.version>
-        <mockito.version>4.11.0</mockito.version>
-        <derby.version>10.14.2.0</derby.version>
+        <mockito.version>5.11.0</mockito.version>
+        <derby.version>10.16.1.1</derby.version>
         <h2.version>2.2.224</h2.version>
         <servlet-api.version>4.0.4</servlet-api.version>
         <log4j.version>2.23.1</log4j.version>
         <slf4j.version>2.0.12</slf4j.version>
-        <commons-io.version>2.15.1</commons-io.version>
+        <commons-io.version>2.16.1</commons-io.version>
         <commons-fileupload.version>1.5</commons-fileupload.version>
         <commons-collections4.version>4.4</commons-collections4.version>
         <snakeyaml.version>2.2</snakeyaml.version>
         <httpclient.version>4.5.14</httpclient.version>
-        <httpcore.version>4.4.13</httpcore.version>
+        <httpcore.version>4.4.16</httpcore.version>
```

Comment:
By the way this version is from 2022 - https://mvnrepository.com/artifact/org.apache.httpcomponents/httpcore
There is a new major version 5 of httpclient with different artifact id and new package names, most probably we have to track separate task to adopt newer apache http client.

Reply:
might give it a try with the next release, the client and core should be updated but will require some refactoring